Added methods for sex determination tool (#40)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.

### Clinical Data Harmonization

> #### Sex Determination

Reviewer: ```suggestion
#### Prediction of participants' sex
```

> We examined the concordance of genetic information with reported gender to identify mislabeled or contaminated samples.

Reviewer: ```suggestion
The clinical metadata provided included a reported gender.
We used genetic data, in concert with the reported gender, to predict participant sex so that we could identify sexually dimorphic outcomes.
This analysis could also reveal samples that may have been contaminated in certain circumstances.
```
@jaclyn-taroni and @jharenza I'd love another pair of eyes on this. Also, we can only get contamination in certain cases, I'd imagine. Where the sex of the contaminating sample or sample handler was different. Should we go into that here, or is it obvious?

Reviewer 2: `We used genetic data, in concert with the reported gender, to predict participant sex so that we could identify sexually dimorphic outcomes.` The prediction done in concert, but independently, then plotted against the reported gender. For that one contaminated sample, it actually never made it to our PBTA release due to being removed for a QC issue, so it was a good check in that it fell outside of our cutoffs for being either male or female. We should note the cutoffs for M/F and I would say something like `within X range (which we should note), we deemed samples as contaminated.`

We used the idxstats utility from SAMTOOLS [@pmid:19505943] to calculate read lengths, the number of mapped reads, and the corresponding chromosomal location for reads to the X and Y chromosomes.
We used the fraction of total normalized X and Y chromosome reads that were attributed to the Y chromosome as a summary statistic.
We reviewed this statistic in the context of reported gender and determined that a threshold of less than 0.2 clearly delineated female samples.
Fractions greater than 0.4 were predicted to be males.

> Samples with values in the range [0.2, 0.4] were marked as unknown.

Reviewer: If there is another mechanism that _also_ suggests that these samples are contaminated, that might not be the best place to discuss filtering due to contamination. An alternative explanation could be that some of these samples are from individuals with more than two sex chromosomes. I don't know that we want to go into that level of detail in this paragraph, but I don't think _this_ is the point where we want to say that the samples are contaminated.

> Samples with values in the range [0.2, 0.4] were marked as unknown.

Reviewer 2: Hmm good point. This sample was really removed due to NGScheckmate.

> We ran this analysis through [CWL](https://github.com/d3b-center/sex-determination-tool) on Cavatica.

Reviewer: URL gives a 404 error.

Reviewer 2: link fixed!

We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.

### Clinical Data Harmonization

> #### Prediction of participants' sex

Reviewer 3: ```suggestion
#### Prediction of participants' genetic sex
```

The clinical metadata provided included a reported gender.

> We used genetic data, in concert with the reported gender, to predict participant sex so that we could identify sexually dimorphic outcomes.

Reviewer 3: ```suggestion
We used DNA data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
```
